helpers/configureImage.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const common = require('./lib/common.js');
+
+const factory = globals => {
+    return function(image, mode='') {
```

Comment:
Might be better to use an options hash here, that way people can say
`configureImage image disable-device-formatting=true disable-device-sizing=true` rather than having to support `configureImage image disable-device-sizing-and-formatting` and whatever other permutations might be required.

Reply:
Thanks! I'll look into that.